browser_use/browser/browser.py

```diff
@@ -194,8 +197,9 @@ async def _setup_standard_browser(self, playwright: Playwright) -> PlaywrightBro
 				'--no-first-run',
 				'--no-default-browser-check',
 				'--no-startup-window',
-				'--window-position=0,0',
-				# '--window-size=1280,1000',
+				f'--window-position={offset_x},{offset_y}',
+				f'--window-size={screen_size["width"]},{screen_size["height"]}',
```

Comment:
```suggestion
				'--start-maximized',
```
Why not just start maximized?

Reply:
@PaperBoardOfficial for context they are only 2 guys and they got nearly 50 thousand stars and hundreds of thousands of messages within the span of a month, while actively fundraising setting up their company. I am working with them to go through all the PRs and give everyone the responses they deserve currently!

Reply:
For a context, I had been contributing to this repo when it had 18k stars. I was actively raising PRs and one fine day Magnus reached out to me and says there's a third party program which will pay me if I contribute to their repo browser use. I said great. I had a online meet with Magnus, he shared the roadmap and said that he'll make me maintainer of the repo and I can raise PRs. I started raising PRs (although never got the maintainer status), and when I reached out to Magnus, that please review my PRs, he ghosted me. The other founder never accepted my friend request on discord despite asking him on github as well. I tried messaging several times. Even the third party program was also stopped because they said if the PRs are not reviewed we can't pay you.
What would a normal person do in this situation? You were asked by the founder to raise PRs and now they are not interested in you. You would simply leave this thing.  I accepted the fact that I am ghosted and left the hope of getting a reply.
And I know they went for fundraising. They weren't replying to my dms, so I had to put out a message in the discord group in front of everyone, then only I got to know that they have gone for fundraising. 
Now, when I have accepted that I am ghosted, then again on one fine day, I get a reply, that hey we are back to building, what normal indian salary (cuz I am indian) would you want. I replied. And now again I am ghosted.
I dont like this cat and mouse game. It would have been great if he didn't have messaged me in the starting at all. 
Nevermind, Its great that you are now onboard. Please feel free to look at any of my PRs (I have 5 open).

Reply:
Ok I understand, I know it sucks when you put in a bunch of work and get no recognition. All I can say is don't take it too personally, no one is trying to ghost you intentionally, there are just tons of competing priorities.